eng/tools/mcp/azure-sdk-java-mcp/src/init-java-sdk.ts

```diff
@@ -0,0 +1,89 @@
+import { CallToolResult } from "@modelcontextprotocol/sdk/types";
+import { findAzureSdkRoot, spawnAsync } from "./utils/index.js";
+
+export async function initJavaSdk(localTspConfigPath?: string, remoteTspConfigUrl?: string): Promise<CallToolResult> {
+    try {
+        let rootDirectory: string = await findAzureSdkRoot(process.cwd());
+        process.chdir(rootDirectory);
+
+        // Run the Java SDK generation command
+        let generateResult;
+        if (localTspConfigPath) {
+            // tsp-client init --debug --tsp-config C:\workspace\azure-rest-api-specs\specification\communication\Communication.Messages\tspconfig.yaml  --commit 123 --repo Azure/azure-rest-api-specs --local-spec-repo C:\workspace\azure-rest-api-specs\specification\communication\Communication.Messages\tspconfig.yaml --save-inputs
+            generateResult = await spawnAsync(
+                "tsp-client",
+                [
+                    "init",
+                    "--debug",
+                    "--tsp-config",
+                    localTspConfigPath,
+                    "--local-spec-repo",
+                    localTspConfigPath,
+                    "--commit",
+                    "123",
```

Comment:
What's 123?

Reply:
The problem is that the commitID is fake. Dev can use it to generate the tsp-location.yaml, and even sync code locally, but it won't work in the PR if they commit it.